include/remill/Arch/Runtime/Intrinsics.h

```diff
@@ -303,4 +305,111 @@ __remill_write_io_port_16(Memory *, addr_t, uint16_t);
 [[gnu::used, gnu::const]] extern Memory *
 __remill_write_io_port_32(Memory *, addr_t, uint32_t);
 
+// More specific hyper calls.
+[[gnu::used, gnu::const]] extern Memory *__remill_x86_set_segment_es(Memory *);
+
+[[gnu::used, gnu::const]] extern Memory *__remill_x86_set_segment_ss(Memory *);
+
+[[gnu::used, gnu::const]] extern Memory *__remill_x86_set_segment_ds(Memory *);
+
+[[gnu::used, gnu::const]] extern Memory *__remill_x86_set_segment_fs(Memory *);
+
+[[gnu::used, gnu::const]] extern Memory *__remill_x86_set_segment_gs(Memory *);
+
+[[gnu::used, gnu::const]] extern Memory *__remill_x86_set_debug_reg(Memory *);
+
+[[gnu::used, gnu::const]] extern Memory *
+__remill_x86_set_control_reg_0(Memory *);
```

Comment:
Set to what?

Reply:
I don't think there's enough information in `__remill_sync_hyper_call` to figure this out.

https://github.com/lifting-bits/remill/blob/master/lib/Arch/X86/Semantics/SYSTEM.cpp#L64-L77